src/libmongoc/doc/learn/get/installing.rst

```diff
@@ -48,7 +152,19 @@ The following Linux distributions provide |libbson| and |libmongoc| packages:
 - `Debian <debian_>`_ and Debian-based distributions, including
   `Ubuntu <debian_>`_ and Ubuntu derivatives, via APT.
 
+.. seealso::
+
+  For a complete list of package names, see: :doc:`/ref/packages`.
```

Comment:
```suggestion
  For a list of available packages, see: :doc:`/ref/packages`.
```
Avoid implying it is a complete list, consistent with the "Important" admonition on the page stating "the information found here may be incomplete or out-of-date".